causalpy/skl_experiments.py

```diff
@@ -362,13 +372,15 @@ def __init__(
         treatment_threshold,
         model=None,
         running_variable_name="x",
+        epsilon: Optional[float] = 0.001,
```

Comment:
If the default is not `None` , the type should not have `Optional`. Remember `Optional[float]` is equivalent to `float | None`

Reply:
Well spotted. I didn't know about that.